### PR TITLE
Blog app - Views 🟢 

### PR DESCRIPTION
**Pull request by haliljon**
## In this PR I was able to:

> - Use preprocessed HTML file with embedded Ruby code.
> - Use layouts and templates for shared content.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,26 @@
...
+    </div>
+</div>
+<div>
+    <h2>Bio</h2>
+    <p> <%= @user.bio%> </p>
+</div>
+<div>
+    <a href="<%= @user.id %>/posts/"><h1> Posts </h1></a>
+    <div>
+        <% @user.posts.last(4).each do |post| %>
```
Comment (afanimartin) [inline app/views/users/show.html.erb]: - Please use the `last_3_posts` method you defined in the User model.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @haliljon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Methods to render latest posts and comments successfully implemented 
- Corresponding views implemented as required. Well done!

# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Find me on slack
### Martin Afani

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (haliljon) [reply app/views/users/show.html.erb]: Hi @afanimartin, I'll do it thank you 🙏

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi John,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉 💯 🌟 

## Highlights 👍
- Good job making recommended changes!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- Let's make our ReadMe standout even better by removing the underlined aspect in the attached screenshot below:

image

Cheers..Happy coding! 💻  🍷  🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**